other-docs/guides/upgrading/v23.md

```diff
@@ -0,0 +1,115 @@
+
+- **Accessibility**: Over 100 fixes across the admin and editor, including standardized tooltips and clearer admin notice prefixes.
+
+For more details check the [WordPress 6.8 release announcement](https://wordpress.org/download/releases/6-8/).
+
+## Headline Features
+
+### Altis Advanced Security, powered by Patchstack
+
+We're excited to launch Altis Advanced Security, a new additional add-on that provides real-time vulnerability protection for your Altis projects. Powered by Patchstack, this integration brings industry-leading virtual patching (vPatching) directly into your Altis environments.
+
+#### Why It Matters
+
+Identifying vulnerabilities is only the first step—mitigating them quickly is crucial. In many organizations, updating plugins or themes can take days or even weeks, leaving your sites exposed in the meantime.
+
+Even more critically, some plugins never release a fix, requiring manual workarounds or firewall rules.
+
+Altis Advanced Security solves this by automatically applying vPatches—plugin-level firewall rules that neutralize known vulnerabilities before an official fix is available. These rules are applied immediately and directly within WordPress, providing fast and targeted protection with minimal performance impact.
+
+#### Key Features
+
+- **Real-Time Vulnerability Mitigation**
+Instantly protects against known threats, without waiting for full plugin updates or release cycles.
+
+- **Smart vPatching**
+Only patches vulnerabilities in plugins you actually have installed, reducing overhead and improving efficiency.
+
+- **Seamless Integration**
+Works alongside Altis's built-in CDN-level WAF to provide protection across Layers 3, 4, and 7.
+
+- **Compliance-Friendly**
+Helps meet security expectations in compliance audits by proactively mitigating risks.
+
+#### What's Next
+
+We’re actively working to expand Altis Advanced Security with new features, including automated anti-virus scanning for uploaded assets.
+
+Altis Advanced Security is available as a paid add-on, billed per site. For pricing and enablement, please contact your Account Manager.
+
+### Altis Local Server Features
+
+This release includes several bug fixes and introduces new capabilities to enhance development workflows. Highlights include:
+
+#### Add Ability to Modify Docker-Compose Configuration 
+
+Altis Local Server now includes an extensible mechanism to allow for package-specific customizations of the `docker-compose` configuration. It enables developers to define additional configurations via Composer packages and have them incorporated into the `docker-compose.yml` generation process. 
+
+Learn more about this feature [here]().
+
+### Altis Dashboard Features
+
+In order to empower and improve developer experience we continue to ship self-service features and improvements to the Dashboard. In this release we are introducing a much expected feature, the ability for you to perform self-service PHP upgrades on your environments:
```

Comment:
I don't think the column as the end is need, it's not quite a list below it. A full stop will be better.

Reply:
Also, I think we should "users" instead of "you" since it states "users" below again in "Now, users can trigger PHP upgrades on their own..."